DOCSP-42256: leftover v2 migration items (#377)
mongodb/docs-golang: source/whats-new.txt

  ``SessionOptions`` type. To learn more, see the
  :ref:`golang-session-txn-options` section of the Transactions guide.

> - The ``Cursor.SetMaxTime()`` method is renamed to

Reviewer: Q: Is this a breaking change since it changes a method name?

>   ``Cursor.SetMaxAwaitTime()``. This method specifies the maximum time
>   that the server waits for new documents when you retrieve documents by
>   using a tailable cursor when querying a capped collection.

Reviewer: S:  This is a pretty wordy sentence. Can it be simplified to something like:
```suggestion
  ``Cursor.SetMaxAwaitTime()``. This method specifies the maximum time
  that the server waits for new documents when retrieving documents with a tailable cursor on a capped collection.
```
